onnx2kerastl/reshape_layers.py

```diff
@@ -174,6 +174,23 @@ def convert_reshape(node, params, layers, lambda_func, node_name, keras_name):
     input_1 = layers[node.input[1]]
 
     if is_numpy(input_1):
+
+        # check reshaping type and validity
+        contains_zero_dim = np.isin(input_1, 0).any()
+        contains_infer_dim = np.isin(input_1, -1).any()
+        dims_to_ignore = None
+        dims_to_keep_unchanged = None
+        if params['allowzero']:
```

Comment:
I think it would be better to switch to `params.get('allowzero')` as `allowzero` may not be in params